batchflow/named_expr.py

```diff
@@ -4,46 +4,11 @@
 
 import numpy as np
 
-
 class _DummyBatch:
     """ A fake batch for static models """
     def __init__(self, pipeline):
         self.pipeline = pipeline
 
-
-def eval_expr(expr, batch=None, pipeline=None, model=None):
```

Comment:
There is no need to remove `eval_expr`. It can take a named expr (or a bunch of exprs) and evaluate it (them) against kwargs given. So you will use `eval_expr` instead a much longer `NamedExpression.eval_expr`

Reply:
I've returned it.